tests/test_provenance_converters.py

```diff
@@ -0,0 +1,174 @@
+"""Tests for PCGL Study to DatasetModel converter."""
+
+from datetime import date
+from pydantic import HttpUrl
+
+from bento_lib.discovery.models.provenance import (
+    Organization,
+    Person,
+)
+from bento_lib.discovery.models.provenance.external.pcgl import (
+    Study,
+    PrincipalInvestigator,
+    Collaborator,
+    FundingSource,
+)
+from bento_lib.discovery.models.provenance.converters.pcgl import (
+    pcgl_study_to_dataset,
+    _parse_participant_criteria,
+)
+
+
+def test_pcgl_study_to_dataset(basic_pi):
+    """Test converting PCGL Study to DatasetModel."""
+    study = Study(
```

Comment:
would prob make sense to load this from an external json file containing a known valid pcgl study json or something, so this file doesn't get super long